static/app/components/events/autofix/autofixSolution.spec.tsx

```diff
@@ -36,165 +38,175 @@ describe('AutofixSolution', () => {
       url: '/issues/123/autofix/update/',
       method: 'POST',
     });
+    jest.mocked(useAutofixRepos).mockReset();
+    jest.mocked(useAutofixRepos).mockReturnValue({
+      repos: [],
+      codebases: {},
```

Comment:
same thing here

Reply:
same here, I couldn't get it to use the mocked api response through `useAutofixData`, and mocking `useAutofixData` directly, `useAutofixRepos` wouldn't use the internal representation for some reason...